package.json

```diff
@@ -30,31 +47,22 @@
     "redux": "^4.0.0",
     "redux-mock-store": "^1.5.3",
     "redux-thunk": "^2.3.0",
+    "sass-loader": "^7.1.0",
     "standard": "^11.0.1",
     "style-loader": "^0.21.0",
-    "yup": "^0.26.2"
-  },
-  "devDependencies": {
-    "babel-core": "^6.26.3",
-    "babel-jest": "^23.0.1",
-    "babel-loader": "^7.1.4",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-2": "^6.24.1",
-    "electron": "^2.0.2",
-    "fetch-mock": "^6.4.4",
-    "jest": "^23.1.0",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
-    "eslint": "^4.19.1",
-    "eslint-plugin-react": "^7.9.1",
-    "nightmare": "^3.0.1",
+    "webpack": "^4.10.2",
     "webpack-cli": "^2.1.4",
+    "webpack-dev-server": "^3.1.4",
     "webpack-livereload-plugin": "^2.1.1",
-    "webpack": "^4.10.2",
-    "webpack-dev-server": "^3.1.4"
+    "yup": "^0.26.2"
   },
   "scripts": {
+    "build": "./node_modules/.bin/webpack -d",
```

Comment:
do you need -d  flag?

Reply:
It's a debug flag. We don't "need" it but for devenv it's easier for debugging. When we build for release we don't use this flag